src/unix.c

```diff
@@ -66,8 +66,12 @@ static int connUnixListen(connListener *listener) {
             serverLog(LL_WARNING, "Failed opening Unix socket: %s", server.neterr);
             exit(1);
         }
+#ifndef SOCK_NONBLOCK
         anetNonBlock(NULL, fd);
+#endif
+#ifndef SOCK_CLOEXEC
```

Comment:
since we do care about CLOEXEC for the listen sockets, maybe we should handle them like NONBLOCK REUSEADDR, and let anetCreateSocket get flags and handle them all internally?

but if we do that, maybe we should avoid bothering to set CLOEXEC if we need an extra system call for it, in for the cases where we don't care (non listen sockets?)
or maybe this is just very old kernels, and we don't care about the extra system call on these..

Reply:
Then we will have to update the definition of `anetUnixServer`. It doesn't seem worth doing it. We can just do it without flags as the function argument, I already know those flags are `SOCK_CLOEXEC` and `SOCK_NONBLOCK`.